ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -27,12 +27,26 @@
     - "{{ connectbox_default_content_root }}"
     - "{{ connectbox_captive_portal_root }}"
 
-# Copy client interface files
-- name: Copy connectbox client interface
+# Copy local client interface files
+- name: Copy local connectbox client interface
+  when: connectbox_client_repo == ""
```

Comment:
Thanks for the review.

So, my reasoning was we would leave the original angular icon interface as a default if the connectbox_client_repo was left blank. If you don't think there is a good case for that, I'm fine with ripping it out and removing the when statements.